next.config.js

```diff
@@ -42,7 +42,8 @@ const initExport = {
       url: process.env.KEYCLOAK_URL,
       redirectUri: process.env.KEYCLOAK_REDIRECT_URI
     },
-    stripePublicApiKey: process.env.STRIPE_PUBLIC_API_KEY
+    stripePublicApiKey: process.env.STRIPE_PUBLIC_API_KEY,
+    enableSPARouting: !(process.env.ENABLE_SPA_ROUTING === "false")
```

Comment:
I find this initially confusing. I can figure out what's going on after unpacking each piece.

Could there be a better way of achieving this?

Reply:
I don't know if this is the time we have to bring in this flow that was setup here https://github.com/reactioncommerce/generator-reaction/blob/master/generators/backend/templates/src/config.js